src/reflect/scala/reflect/internal/Definitions.scala

```diff
@@ -138,6 +145,8 @@ trait Definitions extends api.StandardDefinitions {
     lazy val DoubleTpe    = DoubleClass.tpe
     lazy val BooleanTpe   = BooleanClass.tpe
 
+    lazy val UnitModule   = valueClassCompanion(tpnme.Unit.toTermName)
```

Comment:
`nme.Unit` would be my only nit. (There's pun potential there, I'm sure.)

Reply:
Yes, I confess to the pun, Unit == "You nit!"